schema/release-schema.json

```diff
@@ -25,6 +25,46 @@
       "format": "date-time",
       "omitWhenMerged": true
     },
+    "publisher": {
+      "title": "Publisher",
+      "description": "The original publisher of this release. Use this field to identify the original publisher of a release published as part of a package containing releases from different publishers.",
```

Comment:
Avoid instructions to publishers ("Use this field"). Just describe the semantics. The first clause is sufficient.

Also, since packages will be optional #1084, but bulk JSON lines will become an option, we shouldn't refer to packages even if we keep a second sentence.